src/components/Accordions/__tests__/GeneralAccordion.test.jsx

```diff
@@ -0,0 +1,18 @@
+import { render, cleanup } from '@testing-library/react';
+import GeneralAccordion from '../GeneralAccordion';
+
+describe('FAQ Page', () => {
+  afterEach(() => cleanup());
+
+  test('should render general accordion', () => {
+    const { getByTestId } = render(<GeneralAccordion />);
+    const generalAccordion = getByTestId('valid-generalAccordion');
+    expect(generalAccordion).toBeInTheDocument();
+  });
+
+  test('should render general accordion title', () => {
+    const { getByText } = render(<GeneralAccordion />);
+    const title = getByText('General');
+    expect(title).toBeInTheDocument();
+  });
+});
```

Comment:
Can we move the tests into the folders of the components ? `src/components/accordions/GeneralAccordion` so the component itself and the components test can live inside this folder. Please repeat the same thing with all the accordions and the pages folder too. `src/pages/FAQ/`